src/serverOperations/departmentsServerOperations.js

```diff
@@ -0,0 +1,64 @@
+import { bases } from '../enums';
+import {
+    createRecord,
+    deleteRecord,
+    getRecords,
+    updateRecord,
+} from './crudOperations';
+
+export function departmentsServerOperations(sessionID) {
+    return {
+        createDepartment: async (
+            name,
+            address,
+            contactNumber,
+            actualManager = null,
+        ) => {
+            actualManager = actualManager ? actualManager : undefined;
```

Comment:
Please learn how `||` and `??` work in JS. Then change this line respectively :)